docs/root/operations/cli.rst

```diff
@@ -76,6 +76,12 @@ following are the command line options that Envoy supports.
   `connection` component to run at `trace` level, you should pass ``upstream:debug,connection:trace`` to 
   this flag. See ``ALL_LOGGER_IDS`` in :repo:`/source/common/common/logger.h` for a list of components.
 
+.. option:: --cpuset-threads
+
+  *(optional)* This flag is used to control the number of worker threads if --concurrency option is not
```

Comment:
nit: I assume this is only a Linux thing? Can we potentially make that a little more and possible link to some definition / man page / etc. of cpuset?

Reply:
I added a mention that this is about Linux and a link to the kernel docs.